.template/.rubocop.yml

```diff
@@ -39,14 +39,6 @@ Style/Documentation:
 Style/FrozenStringLiteralComment:
   Enabled: false
 
-# TODO: Enable this rule later
-Style/GlobalVars:
-  Enabled: false
-
-# TODO: Enable this rule later
-Style/RegexpLiteral:
-  Enabled: false
-
```

Comment:
Hi @rosle should we also enable these 2 rules

- Style/FrozenStringLiteralComment
- Naming/FileName:

Reply:
For `Style/FrozenStringLiteralComment`, we can enable 🤔  seems our project rubocop also enable that and it is always auto-fixed after the project is generated.

For `Naming/FileName`, There are errors on the template file e.g. `Gemfile.rb`, `Procfile.dev.rb`, So if we want to enable this rule, I think it is possible, but we will need to exclude those files. 

I'll update and push the changes 👍 ✨